AppCreationScripts/sample.json

```diff
@@ -12,15 +13,20 @@
   "AADApps": [
     {
       "Id": "service",
-      "Name": "TodoListService",
+      "Name": "TodoListService-Cert",
       "IsPublicClient": false,
+      "Kind":"WebApi",
+      "AvailableToOtherTenants": false,
```

Comment:
See ... **My Org**

you might want to set **Audience** to **AzureADMyOrg**.  This will generate the right things